### PR TITLE
Fishing-related bounties.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
This PR adds three basic/assistant bounties for shipping fish. The reward is above par when compared to other basic bounties, however, each shipped fish that is either dead or comes from a case (bought from cargo, cause that's lazy) will reduce the payout by a portion of it, also the pool of assistant bounties is quite big you wouldn't roll them too often.

The three bounties are the following: One for any kind of fish, another for fish of matching fluid type (freshwater, saltwater, sulphuric water), and lastly, one for specific types of fish, possibly easy money if RNG is with you.

## Why It's Good For The Game
I wanted to add something diffrent from most grimy assistant/basic bounties where you either just print the stuff from the nearby autolathe or fetch it for little money. Also, this provides another outlet to fishing, I guess.

## Changelog

:cl:
add: Added a few fish related bounties.
add: Fish cases to store and preserve life fish within can be now printed from the service techfab and the autolathe.
/:cl:

### CODE REVIEW

**Comment by Cadunkus** [comment]
Now to fix fishing so it isn't complete trash!

File: code/modules/cargo/bounties/assistant.dm [hunk 1]
```diff
@@ -209,3 +209,79 @@
...
+///A subtype of the fish bounty that requires specific fish types. The higher their rarity, the better the pay.
+/datum/bounty/item/assistant/fish/specific
+	description = "Our prestigious fish collection is currently lacking a few specific species. Fishes that are dead or from fish cases will only be paid half as much."
+	reward = CARGO_CRATE_VALUE * 16
+	required_count = 3
+	wanted_types = list()
+
+/datum/bounty/item/assistant/fish/specific/New()
+	var/static/list/choosable_fishes
+	if(!choosable_fishes)
```
Comment (Jacquerel) [inline code/modules/cargo/bounties/assistant.dm:262]:
```suggestion
	if(isnull(choosable_fishes))
```

**Review by Jacquerel — CHANGES_REQUESTED** [review]
How do you deliver a live fish? Just do it quickly? Or is there some equipment I have forgotten about which preserves them?

**Comment by Ghommie** [comment]
> How do you deliver a live fish? Just do it quickly? Or is there some equipment I have forgotten about which preserves them?

Fish cases, though it seems there's currently only one way to get them and that is buying fish from cargo. Guess I'll have to add a design so that they can be printed from lathes.

**Pull request merged**